frontend/packages/integration-tests-cypress/tsconfig.json

```diff
@@ -1,9 +1,9 @@
+
 {
-  "extends": "../../tsconfig",
   "compilerOptions": {
     "types": ["cypress"]
   },
   "include": [
-    "**/*.ts"
+    "../../node_modules/cypress", "**/*.ts"
   ]
```

Comment:
This doesn't seem quite right. Should this be like the recommendation found here: https://docs.cypress.io/guides/tooling/typescript-support#Configure-tsconfigjson

Reply:
Yeah, I remember tinkering with this a while back and it was very temperamental. My gut says that we shouldn't need to directly include Cypress in our build like this. But if it works, we can revisit it later.